redhat-cop/containers-quickstarts
# Add CI tests to this repo.

#### What is this PR About?
Adds a travis integration to test this repository.

addresses #167 #157 #82 

#### How do we test this?

* Read through the new TESTING.md doc, follow instructions to test locally.
* See travis run at https://travis-ci.org/etsauer/containers-quickstarts/

NOTE: There are broken builds in this repo. The tests currently fail because of them. I am not going to fix them in this PR. We can either fix them separately in a PR before we merge this, or do a followup.

cc: @redhat-cop/day-in-the-life
